src/controllers/midi/midisourceclock.cpp

```diff
@@ -0,0 +1,117 @@
+#include "controllers/midi/midimessage.h"
+#include "controllers/midi/midisourceclock.h"
+#include "util/math.h"
+
+bool MidiSourceClock::handleMessage(unsigned char status) {
+    switch (status) {
+    case MIDI_START:
```

Comment:
supporting midi_continue is going to be tricky because right now the midisourceclock handles bpm calculation, but beat fraction information is handled by the calling code.  It will take some work to figure out how to freeze the data, then thaw it, because all the timing is based on an absolute clock.  For now this will be a todo.

Since this code won't be exposed in Mixxx, I think it's safe to have it be somewhat feature incomplete.